src/routes/components/grid/+page.svelte

```diff
@@ -1,289 +1,125 @@
-				</Col>
-			{/each}
-		</Row>
-		<Row noGutters>
-			<Col cols="8">
-				<Card class="card-example">col-8</Card>
-			</Col>
-			<Col cols="4">
-				<Card class="card-example">col-4</Card>
-			</Col>
-		</Row>
-		<Row noGutters>
-			{#each [1, 2, 3, 4] as col}
-				<Col>
-					<Card class="card-example">col</Card>
-				</Col>
-			{/each}
-		</Row>
-	</Container>
+<p>
+	By default, flex components will automatically fill the available space in a row or column. They will also shrink
+	relative to the rest of the flex items in the flex container when a specific size is not designated. You can define
+	the column width of the col by using the cols prop and providing a value from 1 to 12.
+</p>
+<Preview src="./GridGrow.svelte">
+	<GridGrow />
 </Preview>
 
 <h2>Nested grid</h2>
-<p>Nested grid</p>
-<Preview>
-	<Container class="container-example">
-		<Row noGutters>
-			<Col cols="9">
-				<Card class="card-example">Level 1: .col-sm-9</Card>
-				<Row noGutters>
-					<Col cols="8">
-						<Card style="background-color: lightgrey;" class="card-example">Level 2: .col-8</Card>
-					</Col>
-					<Col cols="4">
-						<Card style="background-color: lightgrey;" class="card-example">Level 2: .col-4</Card>
-					</Col>
-				</Row>
-			</Col>
-		</Row>
-	</Container>
+<p>Grids can be nested, similar to other frameworks, in order to achieve very custom layouts.</p>
+<Preview src="./GridNested.svelte">
+	<GridNested />
 </Preview>
 
 <h2>One column width</h2>
 <p>One column width</p>
-<Preview>
-	<Container class="container-example">
-		<Row noGutters>
-			{#each [1, 2, 3] as col}
-				<Col cols={col == 2 ? 6 : undefined}>
-					<Card class="card-example">{col} of 3 {col == 2 ? '(wider)' : ''}</Card>
-				</Col>
-			{/each}
-		</Row>
-		<Row noGutters>
-			{#each [1, 2, 3] as col}
-				<Col cols={col == 2 ? 5 : undefined}>
-					<Card class="card-example">{col} of 3 {col == 2 ? '(wider)' : ''}</Card>
-				</Col>
-			{/each}
-		</Row>
-	</Container>
+<Preview src="./GridColumnWidth.svelte">
+	<GridColumnWidth />
 </Preview>
 
 <h2>Row and column breakpoints</h2>
-<p>Row and column breakpoints</p>
-<Preview>
-	<Container class="container-example">
-		<Row noGutters>
-			{#each [1, 2] as col}
-				<Col :lg="cols[n - 1]" :md="6" :sm="cols[n - 1]">
-					<Card class="card-example">{col} of 3 {col == 2 ? '(wider)' : ''}</Card>
-				</Col>
-			{/each}
-		</Row>
-		<Row noGutters>
-			{#each [1, 2, 3] as col}
-				<Col cols="sm">
-					<Card class="card-example">col</Card>
-				</Col>
-			{/each}
-		</Row>
-	</Container>
+<p>
+	Dynamically change your layout based upon resolution. (resize your screen and watch the top row layout change on sm,
+	md, and lg breakpoints)
+</p>
+<Preview src="./GridRow.svelte">
+	<GridRow />
+</Preview>
+<h2>Container</h2>
+<p>Container breakpoints</p>
+<Preview src="./GridRow.svelte">
+	<GridContainer />
 </Preview>
 
 <style global>
 	h2 {
-		margin-top: 2rem;
+		margin-top: 3rem !important;
 	}
 	.u-preview-body {
```

Comment:
We can add a prop for Preview component to set it's direction to vertical or horizontal. and also Preview should have class prop and we add example-* classes to it.

```
<Preview class="container-example" vertical src="./GridDefault.svelte">
   ...
</Preview>
```

what do you think?

Reply:
I updated Preview in https://github.com/ubeac/svelte/pull/419 and added these features :)